### PR TITLE
[TFLite] Add int16x8 support for the MIRROR_PAD operator

### PR DESCRIPTION
**Pull request by johan-gras**
Hello,

This PR adds int16x8 support for the MIRROR_PAD operator.

Thanks,
Johan.

### CODE REVIEW

**Comment by gbaned** [comment]
@jianlijianli  Can you please review this PR ? Thanks!

**Comment by gbaned** [comment]
@jianlijianli Can you please review this PR ? Thanks!

**Comment by gbaned** [comment]
@johan-gras  Can you please sign CLA. Thank you!

**Comment by gbaned** [comment]
@johan-gras Can you please sign CLA. Thank you!

**Comment by johan-gras** [comment]
@gbaned signed.

**Comment by gbaned** [comment]
Hi @johan-gras Can you please resolve conflicts? Thank you!

**Comment by abattery** [comment]
@alankelly in terms of quantized kernel implementation wise, it looks okay to me.

Could you review this change as TFLite op/kernel perspective?

**Comment by gbaned** [comment]
Hi @johan-gras Can you please resolve conflicts? Thank you!

**Comment by Tessil** [comment]
@gbaned Thanks, I resolved the conflicts.

File: tensorflow/lite/kernels/mirror_pad.cc [hunk 1]
```diff
@@ -272,6 +276,19 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
...
+  if (input_tensor->type == kTfLiteUInt8 || input_tensor->type == kTfLiteInt8 ||
+      input_tensor->type == kTfLiteInt16) {
+    TF_LITE_ENSURE_EQ(context, input_tensor->params.scale,
+                      output_tensor->params.scale);
+    TF_LITE_ENSURE_EQ(context, input_tensor->params.zero_point,
+                      output_tensor->params.zero_point);
+  }
+
+  if (input_tensor->type == kTfLiteInt16) {
+    TF_LITE_ENSURE_EQ(context, input_tensor->params.zero_point, 0);
```
Comment (alankelly) [inline tensorflow/lite/kernels/mirror_pad.cc:288]: Same comment I made TILE, do we care what the zero points are, once they are equal?

File: tensorflow/lite/tools/versioning/runtime_version.cc [hunk 2]
```diff
@@ -280,6 +280,7 @@ std::string FindMinimumRuntimeVersionForOp(tflite::BuiltinOperator op_code,
            {{BuiltinOperator_SQUARED_DIFFERENCE, 2}, "2.5.0"},
            {{BuiltinOperator_MIRROR_PAD, 1}, "1.13.1"},
            {{BuiltinOperator_MIRROR_PAD, 2}, "2.3.0"},
+           {{BuiltinOperator_MIRROR_PAD, 3}, "2.11.0"},
```
Comment (alankelly) [inline tensorflow/lite/tools/versioning/runtime_version.cc:283]: 2.12.0

Comment (Tessil) [reply tensorflow/lite/tools/versioning/runtime_version.cc]: Thanks, updated it.

**Review by alankelly — APPROVED** [review]

**Pull request merged**